analytics-listeners/src/main/java/org/exoplatform/analytics/listener/portal/PageAccessListener.java

```diff
@@ -101,6 +103,10 @@ private StatisticData buildStatisticData(WebuiRequestContext context) {
       statisticData.addParameter("userLocale", portalRequestContext.getLocale());
       statisticData.addParameter("portalOwner", portalRequestContext.getPortalOwner());
       statisticData.addParameter("portalUri", portalRequestContext.getPortalURI());
+      statisticData.addParameter("pageTitle", portalRequestContext.getTitle());
+      if (space != null) {
+        statisticData.addParameter("spaceId", space.getId());
+      }
```

Comment:
```suggestion
```
No need to add the spaceId since it's already 
`      addSpaceStatistics(statisticData, space);`
`    statisticData.setSpaceId(Long.parseLong(space.getId()));`